packages/apps/plugins/plugin-chess/src/components/ChessMain.tsx

```diff
@@ -37,7 +37,7 @@ export const ChessMain = ({ data: object }: { data: Game }) => {
   }
 
   return (
-    <Main.Content classNames={['flex flex-col min-bs-[calc(100dvh-2.5rem)] overflow-hidden', coarseBlockPaddingStart]}>
+    <Main.Content classNames={[baseSurface, fixedInsetFlexLayout, coarseBlockPaddingStart]}>
```

Comment:
`Main.Content` should not have a background color in principle — instead this should be applied to `html { background-color: … }` in `theme.css`